llama.cpp

```diff
@@ -2072,16 +2072,18 @@ struct llama_vocab {
     id special_cls_id  = -1;
     id special_mask_id = -1;
 
-    int special_add_bos = -1; // -1 unknown, 1 add, 0 don't add.
-    int special_add_eos = -1; // -1 unknown, 1 add, 0 don't add.
-
     id linefeed_id       = 13;
     id special_prefix_id = -1;
     id special_suffix_id = -1;
     id special_middle_id = -1;
     id special_eot_id    = -1; // TODO: move above after "eos_id", and here add "file separator" token
 
-    bool add_space_prefix = true;
+    // tokenizer flags
+    bool tokenizer_add_space_prefix = true;
+    bool tokenizer_special_add_bos  = false;
+    bool tokenizer_special_add_eos  = false;
+    bool tokenizer_ignore_merges    = false;
+    bool tokenizer_mask_lstrip      = false;
```

Comment:
These should probably be also exported during conversion, similar to `add_bos` and `add_eos`?